dev/KaairaGupta/data_loader.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
+
+
+
+def load_split_preprocessed_data(i):
+
+    data = pd.read_csv("../../datasets/winequality.csv")
+    Count_Fraud_transacation = len(data["recommend"]==True) 
```

Comment:
Please adhere to python naming conventions as per PEP8 (this may also be enforced by black)

Reply:
Yes, sorry for this. Made changes in the entire files.